build.sbt

```diff
@@ -817,11 +826,14 @@ lazy val bench = project.in(file("test") / "benchmarks")
       else "org.scala-lang" % "scala-compiler" % benchmarkScalaVersion :: Nil
     },
     //scalacOptions ++= Seq("-feature", "-opt:inline:scala/**", "-Wopt"),
-    scalacOptions ++= Seq("-feature", "-opt:l:inline", "-opt-inline-from:scala/**", "-opt-warnings"),
+    scalacOptions ++= Seq("-feature", "-opt:l:inline", "-opt-inline-from:scala/**", "-opt-warnings")
```

Comment:
the trailing comma was removed. Also the commented option should be used by now. Somehow that change never gets in...

Reply:
I reverted the trailing comma.